tests/test_0001_html.py

```diff
@@ -257,7 +257,8 @@ def test_attribute_list():
 
 
 def test_html_strings():
-    from lona.html import HTML, Button
+    from lona.html import HTML, Button, TextInput, TextArea, CheckBox, \
```

Comment:
please use parentheses in cases like this to keep it consistent with the rest of the code base

Reply:
Fixed